spec/views/accounts/show.html.erb_spec.rb

```diff
@@ -111,6 +111,38 @@
     end
   end
 
+  context 'when current user has an in_person_enrollment that was failed' do
+    let(:vtr) { ['Pe'] }
+    let(:sp_name) { 'sinatra-test-app' }
+    let(:user) { build(:user, :with_pending_in_person_enrollment) }
+
+    before do
+      # Make the in_person_enrollment and associated profile failed
+      in_person_enrollment = user.in_person_enrollments.first
+      in_person_enrollment.update!(status: :failed, status_check_completed_at: Time.zone.now)
+    end
```

Comment:
@eileen-nava  For both, we need to update the profile as well. (You did this for expired below)

```      
profile = user.profiles.first
profile.deactivate_due_to_in_person_verification_cancelled
```

Reply:
@gina-yamada Thanks for catching that!  I fixed it in [9e19f4a](https://github.com/18F/identity-idp/pull/11141/commits/9e19f4a993b2aa3e75192f4a927963e46ef19ce5).

Reply:
Thanks for adding these specs.